File: VisitBosnia/visit_bosnia_mobile/lib/model/attractions/attraction_search_object.dart
class AttractionSearchObject {
  String? searchText;
  bool? includeIdNavigation;
  int? categoryId;
  int? cityId;

  AttractionSearchObject(
      {this.searchText, this.includeIdNavigation, this.categoryId});

  AttractionSearchObject.fromJson(Map<String, dynamic> json) {
    searchText = json['searchText'];
    includeIdNavigation = json['includeIdNavigation'];
    categoryId = json['categoryId'];
    cityId = json['cityId'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['searchText'] = this.searchText;
    data['includeIdNavigation'] = this.includeIdNavigation;
    data['categoryId'] = this.categoryId;
    data['cityId'] = this.cityId;
    return data;
  }
}

File: VisitBosnia/VisitBosnia.Model/SearchObjects/AttractionSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.Requests
{
    public class AttractionSearchObject 
    {
        public string? SearchText { get; set; }
        public bool IncludeIdNavigation { get; set; }
        public int? CityId { get; set; }
        public int? CategoryId { get; set; }
    }
}
